### PR TITLE
update standalone CLI doc

### PR DESCRIPTION
**Pull request by rmgogogo**
Update docs for standalone CLI.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1416)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1416--competent-brattain-de2d6d.netlify.com/docs/pipelines/standalone-deployment-gcp/

File: content/docs/pipelines/standalone-deployment-gcp.md [hunk 1]
```diff
@@ -168,6 +171,14 @@ You can uninstall Kubeflow Pipelines by:
     # kubectl delete -k manifests/kustomize/env/gcp
     ```
 
+## Optionally delete cluster
+
+If you don't want the cluster in the future, run following command to delete it.
```
Comment (sarahmaddox) [inline content/docs/pipelines/standalone-deployment-gcp.md]: s/If you don't want the cluster in the future/If you no longer need the cluster

File: content/docs/pipelines/standalone-deployment-gcp.md [hunk 2]
```diff
@@ -51,19 +51,17 @@ See the Google Kubernetes Engine (GKE) guide to [configuring cluster access for
...
 
     gcloud container clusters create $CLUSTER_NAME \
         --zone $ZONE \
         --machine-type $MACHINE_TYPE \
-        --scopes $SCOPES \
-        --num-nodes 2 \
-        --max-nodes 5 \
-        --min-nodes 2 \
-        --enable-autoscaling
+        --scopes $SCOPES
```
Comment (Bobgy) [inline content/docs/pipelines/standalone-deployment-gcp.md]: I'm curious why disable auto scaling?

File: content/docs/pipelines/standalone-deployment-gcp.md [hunk 3]
```diff
@@ -51,19 +51,17 @@ See the Google Kubernetes Engine (GKE) guide to [configuring cluster access for
...
     # The following parameters can be customized based on your needs.
 
+    gcloud config set project <your-gcp-project>
+
     CLUSTER_NAME="kubeflow-pipelines-standalone"
     ZONE="us-central1-a"
-    MACHINE_TYPE="n1-standard-2" # A machine with 2 CPUs and 7.50GB memory
-    SCOPES="storage-rw,cloud-platform" # These scopes are needed for running some pipeline samples
+    MACHINE_TYPE="n1-standard-2" # Minimum required resources
+    SCOPES="cloud-platform" # Allow talks with GCP APIs without additional settings
```
[the remark this answers is not in the document]
Comment (Bobgy) [reply content/docs/pipelines/standalone-deployment-gcp.md]: I think it was my mistake to set scope like this when it wasn't used at all. Shall we drop this scope line?

File: content/docs/pipelines/standalone-deployment-gcp.md (hunk 2 above)
Comment (rmgogogo) [reply content/docs/pipelines/standalone-deployment-gcp.md]: Default it to align with best GKE practice.
Actually the GKE auto-scale also can't work with KFP typed workload.

Comment (Bobgy) [reply content/docs/pipelines/standalone-deployment-gcp.md]: Thanks, make sense to me

**Comment by sarahmaddox** [comment]
@rmgogogo Please would you take a look at the remaining comments on this PR?

**Comment by sarahmaddox** [comment]
@rmgogogo Shall we close this PR?

**Comment by Bobgy** [comment]
@Renmin Gu <renming@google.com>

Sarah Maddox <notifications@github.com> 于 2020年1月30日周四 下午1:30写道：

**Comment by sarahmaddox** [comment]
/close

I'm closing this PR due to lack of activity. Please do re-open it if there's still work ongoing.

**Pull request closed without merging**